pypsi/shell.py

```diff
@@ -180,6 +181,8 @@ def set_readline_completer(self):
             readline.parse_and_bind("tab: complete")
             self._backup_completer = readline.get_completer()
             readline.set_completer(self.complete)
+            if self.completer_delims is not None
```

Comment:
Looks like you're missing a `:` here. Add that and I think this is good to merge.